lib/commands/doctor/checks/logged-in-user.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+const execa = require('execa');
+const errors = require('../../../errors');
+const chalk = require('chalk');
+
+const taskTitle = 'Checking logged in user';
+
+function loggedInUser() {
+    const uid = process.getuid();
+    let ghostuid;
+
+    try {
+        ghostuid = execa.shellSync('id -u ghost').stdout;
+    } catch (e) {
+        // CASE: the ghost user doesn't exist yet, all good in the hood
+        return;
+    }
+
+    ghostuid = parseInt(ghostuid);
+
+    if (ghostuid === uid) {
+        throw new errors.SystemError({
+            message: 'You can\'t install Ghost with a user called `ghost`. Please use a different user(name).',
+            help: `${chalk.green('https://docs.ghost.org/docs/install#section-create-a-new-user')}`,
+            task: taskTitle
+        });
+    }
+
+    return;
+}
+
+module.exports = {
+    title: taskTitle,
+    task: loggedInUser,
+    enabled: (ctx) => !ctx.local && !(ctx.instance && ctx.instance.process.name === 'local') && ctx.system.platform.linux,
```

Comment:
might be worth also checking if `argv.process === 'local'` -> that way if you do `ghost install --process local`, this check will be skipped.

Reply:
Good call!!! Changes are pushed! Ready for 🚀